### DISCUSSION TITLE
Interpret minor as major

### DISCUSSION THREAD

**Discussion by SimonCockx** [renovatebot/renovate#18935] [Request Help] [answered]
We use a package called Xtext which consistently introduces backwards-incompatible changes in its minor versions. Is there a way to say to renovate that it should treat a minor version increase as a major one?

E.g., a minor Xtext upgrade should match this rule:
```
"major": {
    "dependencyDashboardApproval": true
}
```
but it should not match this rule:
```
{
    "matchPackagePatterns": ["*"],
    "matchUpdateTypes": [
        "minor",
        "patch"
    ],
    "groupName": "all non-major dependencies",
    "groupSlug": "all-minor-patch"
}
```

**Comment by rarkins** [comment, marked as answer]
major, minor and patch are defined terms which we don't want to redefine for certain packages. Really what you're concerned about is breaking vs non-breaking. We have a proposal for handling this type of thing as described in https://github.com/renovatebot/renovate/issues/1257#issuecomment-1102362352

For now you should define a second package rule for the packages which make breaking changes in minor packages to remove groupName/groupSlug so they are not grouped
